Makefile

```
@@ -43,6 +43,10 @@ _up-%:
 		--attach=$* \
 		$*
 
+up-php-8-4: _up-php-8-4
```

checkmake (0.3.2)

[warning] Target "up-php-8-4" should be declared PHONY.

(phonydeclared)